src/environmentd/Cargo.toml

```diff
@@ -29,6 +29,7 @@ http-body = "0.4.5"
 humantime = "2.1.0"
 hyper = { version = "0.14.23", features = ["http1", "server"] }
 hyper-openssl = "0.9.2"
```

Comment:
for context, I spent a lot of yesterday hitting openssl/tls issues when trying to use the `HTTPSConnector` provided by `hyper-openssl`, but then realized that we use `hyper-tls` for the same thing in `mz-ore` which is why that's included here, and it worked out of the box.